README.md

```diff
@@ -12,21 +12,54 @@ do so, please follow the steps below. Those are actually based off
 
 ```shell
 $ pip install sphinx sphinx-autobuild sphinx_rtd_theme sphinx-intl
+Collecting sphinx
```

Comment:
I know this is just highlighting the expected return of the commands, but it fells odd to me to see it put like this. 
For an inexperienced Linux user this might look like part of the command he has to copy to his terminal. Maybe highlighting the command somehow might help

Reply:
There is a warning while generating the documentation indicating that, once the dollar sign is added to the command, there should be some output.

Reply:
I'll remove everything - including the $.

Reply:
Done